config/v2_0/types/unit_test.go

```diff
@@ -40,7 +41,7 @@ func TestSystemdUnitValidate(t *testing.T) {
 		},
 		{
 			in:  in{unit: SystemdUnit{Contents: "[Foo"}},
-			out: out{err: errors.New("invalid unit content: unable to find end of section")},
+			out: out{err: fmt.Errorf("invalid unit content: unable to find end of section")},
```

Comment:
If generator functions for adding custom bits to the errors is the way we want to move the `invalid unit content: %q` error could be moved to the error package as well.